setup.cfg

```diff
@@ -32,3 +32,36 @@ match-dir = ^(?!(ci|cpp|conda|docs|java|notebooks)).*$
 ignore-decorators = .*(docutils|doc_apply|copy_docstring).*
 select = 
     D30
+
+[mypy]
```

Comment:
Curious is the "order" of which config file to use:

1. setup.cfg
2. pyproject.toml
3. specific config

Reply:
Gotcha. Totally makes to centralize the configs first and then transfer the configs to the preferred file format.

Agreed in the pyproject.toml > setup.cfg > others ordering too. We use the same reasoning in pandas too :)